apps/ui/src/components/providers/ThemeProvider.tsx

```diff
@@ -48,6 +48,18 @@ export const pcglHeaderTheme: ThemeConfig = {
 	},
 };
 
+export const pcglFooterTheme: ThemeConfig = {
+	components: {
+		Layout: {
+			footerBg: '#520339',
+		},
+		Typography: {
+			colorText: '#FFFFFF',
+			colorLink: '#FFD6E7',
+			colorLinkHover: 'rgba(255, 214, 231, 0.5)',
+		},
+	},
+};
```

Comment:
it's good to add blank lines between different functions/objects, as it facilitates reading
```suggestion
};

```